frontend/uttarakhand-floods-2013/style.css

```
@@ -0,0 +1,1327 @@
+body.light-theme .data-table th[scope="row"] {
+    color: #0369a1;
+}
+
+.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)